configs/test/bot/setup/android.bash

```diff
@@ -13,6 +13,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function run_bot () {
+  bot_directory=$INSTALL_DIRECTORY/bots/$(echo $1 | sed s/:/-/)
+  mkdir -p $bot_directory
+  cp -r $INSTALL_DIRECTORY/clusterfuzz $bot_directory/clusterfuzz
+  echo "Created bot directory $bot_directory."
+
+  while true; do
+    if ! $("$ADB_PATH/adb" -s "$1" wait-for-device); then
+      echo "Device $1 is not connected."
+      exit 1
```

Comment:
should we exit or wait?

Reply:
Since the command above is wait-for-device I'm not actually sure if we can get into this state. I'm actually just going to remove it, since this only made sense in the older code it was based on.